flytekit/types/directory/types.py

```diff
@@ -17,6 +17,7 @@
 from flytekit.models.types import LiteralType
 
 T = typing.TypeVar("T")
+PathType = typing.Union[str, os.PathLike]
```

Comment:
aliased the type here to keep line length within black constraints to avoid wrapped lines - as not sure `# type :ignore` comment works  for multi line expressions